samples/keycloak-pkce-cmdline-sample/README.md

```diff
@@ -0,0 +1,36 @@
+# Instructions for the Keycloak OAuth2 with PKCE Command-Line Sample
+
+## Browse Online
+
+[Browse Source][browse-source], or main file [DailyMotionSample.java][main-source].
+
+## Command-Line Instructions
+
+**Prerequisites:** install [Java 6 or higher][install-java], [git][install-git], and
+[Maven][install-maven]. You may need to set your `JAVA_HOME`.
+
+1. Check out the sample code:
```

Comment:
Not sure what keycloak is, or if we can include this part in a google project

Reply:
ahh... that had not occurred to me. Keycloak is a JBoss (Red Hat) project, it's an "open source identity and access management solution" (https://www.keycloak.org/) and it's released under the Apache 2.0 license.
So, as such there shouldn't be a problem using it, especially since it's only being used for the sample demonstration and not as an actual component. But maybe there is some other more political/legal reason for it not being possible. I could remove the sample but I would like to keep it as it's pretty good and clear for the purpose of testing and demonstration.

Reply:
Not a license issue, but we would like the docs for this product to be self-contained.

Reply:
gotcha. However, the daily motion example isn't exactly self-contained is is? It actually depends on the user registering an account with dailymotion.com.

But if using keycloak is not acceptable, can you suggest an alternative or should I remove the sample completely?

Reply:
@elharo any thoughts on this?